AirCastingTests/SearchAndFollowTests/SessionForLocationDownloaderTests.swift

```diff
@@ -0,0 +1,87 @@
+// Created by Lunar on 02/08/2022.
+//
+import XCTest
+import Resolver
+@testable import AirCasting
+
+class SessionForLocationDownloaderTests: ACTestCase {
+    lazy var sut = SessionsForLocationDownloaderDefault()
+    let clientSpy = APIClientSpy()
+    
+    func test_whenRequestingDataFromServer_shouldUseGetMethod() {
+        Resolver.test.register { self.clientSpy as APIClient }
+        MockedSession.getAnySession(using: sut)
```

Comment:
Hm, I'm not sure about making it a func of MockedSession. Maybe it can be changed to a function `requestSessions()` that would be a method of SessionForLocationDownloaderTests and it would call `sut.requestSessions(....)` with some mocked data? I'm not sure about this MockSessions struct, as it is not needed for anything else that calling this function and it's storing geoSquare as well, which is no a part of a sessions.